extensions/interactions/MusicNotesInput/directives/MusicNotesInput.js

```diff
@@ -334,43 +360,61 @@ oppia.directive('oppiaInteractiveMusicNotesInput', [
           var noteChoicesDiv = element.find('.oppia-music-input-note-choices');
 
           for (var i = 0; i < scope.noteSequence.length; i++) {
-            var innerDiv = $('<div></div>')
-              .data('noteType', NOTE_TYPE_NATURAL)
-              .data('noteId', scope.noteSequence[i].note.noteId)
-              .addClass('oppia-music-input-natural-note')
-              .addClass('oppia-music-input-on-staff')
-              .draggable({
-                // Keeps note from being placed on top of the clef.
-                containment: '.oppia-music-input-valid-note-area',
-                cursor: 'pointer',
-                stack: '.oppia-music-input-note-choices div',
-                grid: [scope.HORIZONTAL_GRID_SPACING, 1],
-                start: function() {
-                  $(this).data('leftPosBeforeDrag', $(this).position().left);
-                },
-                revert: function() {
-                  var draggableOptions = $(this);
-                  // If note is out of droppable or off staff, remove it.
-                  if (isCloneOffStaff(draggableOptions)) {
-                    scope._removeNotesFromNoteSequenceWithId(
-                      draggableOptions.data('noteId'));
-                    scope._sortNoteSequence();
-                    draggableOptions.remove();
+            if (scope.interactionIsActive) {
+              var innerDiv = $('<div></div>')
```

Comment:
As above, do what you can to reduce duplication.

Reply:
Ok, will change those.